scripts/build_deb

```diff
@@ -50,7 +50,7 @@ function main()
 
 function validate_arguments()
 {
-    [[ $DISTRIB =~ ^stretch$ ]] || critical "Invalid distribution $DISTRIB"
+    [[ $DISTRIB =~ ^buster|stretch$ ]] || critical "Invalid distribution $DISTRIB"
```

Comment:
will stretch be supported ? I suppose this will be removed soon or later ?

Reply:
I would say that building both for a while could be good thing.